### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by cxtruong70**
Made minor edits to improve readability by removing adverbs and hedging statements like "manually" and "... intended."

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,15 +12,15 @@ To install it, please run:
 python3 -m pip install web.py==0.62
 ```
 
-If you are still using Python 2.7, then please use web.py version 0.51
-which is intended to be our last release that supports Python 2. 
+If you are using Python 2.7, then please use web.py version 0.51
```
Comment (cclauss) [inline README.md:15]:
```suggestion
If you are still using Python 2.7, then please use web.py version 0.51
```
The use of `still` was quite deliberate and is even more appropriate today then when it was first written.

File: README.md [hunk 2]
```diff
@@ -12,15 +12,15 @@ To install it, please run:
...
+which is our last release that supports Python 2. 
 ```
 # For Python 2.7
 python2 -m pip install web.py==0.51
 ```
 
 You can also download it from [GitHub Releases](https://github.com/webpy/webpy/releases)
```
Comment (cclauss) [inline README.md:22]: Now that pip contains a real dependency resolver, I no longer see any advantage in recommending this method of installation.

**Review by cclauss — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (cxtruong70) [reply README.md:22]: Good point. We should remove it entirely as the user should never run into dependencies issues related to this.

**Review by cxtruong70 — COMMENTED** [review]
I have removed the recommendation for Python 2.7 as discussed.

**Review by iredmail — APPROVED** [review]

**Comment by cclauss** [comment]
@iredmail Do you understand why tests are failing?

**Comment by iredmail** [comment]
Seems caused by MySQL default charset `utf8mb4`.
FYI https://stackoverflow.com/questions/43437490/pdo-construct-server-sent-charset-255-unknown-to-the-client-please-rep

**Comment by cclauss** [comment]
https://dev.mysql.com/doc/relnotes/mysql/8.0/en/

**Review by cclauss — APPROVED** [review]
Thanks for doing this.

**Pull request merged**